packages/terra-core-docs/src/terra-dev-site/doc/icon/AccessabilityGuide.2.doc.mdx

```diff
@@ -0,0 +1,111 @@
+
+### Code Considerations
+
+- Appropriately code each icon as being meaningful or decorative. Ensure each meaningful icon has alternative text provided by the content creator.
+- Do not overwrite theme colors unless UX provides custom colors. All custom colors must pass WCAG 2.1 color accessibility requirements.
+
+<Whitespace />
+
+### Content Considerations
+
+- Specify **every** icon instance as meaningful or decorative. Provide developers with clear and concise alternative text for meaningful icons that accurately reflect the icon's meaning or purpose.
+- While some icons are ubiquitous and generally intuitive (e.g., printer icon), it is best to use words with icons.
+- Icon meaning must be consistent throughout the product. For example, a plus (+) icon must always represent “add” functionality. In addition, if multiples of the same icon are in [repeated controls on the same page](https://adrianroselli.com/2019/06/group-labels-do-not-guarantee-uniquity.html), the combination of alternative text and text label must be [unique](https://fae.disability.illinois.edu/rulesets/CONTROL_10/) (e.g., “Add Blood Pressure” and “Add Temperature”).
+- Colors for icons are predefined in each of Terra's visual themes. When icon colors require customization, they must meet WCAG Success Criteria for [1.4.3 Contrast (Minimum) (Level AA)](https://www.w3.org/WAI/WCAG21/Understanding/contrast-minimum.html), which is a 3:1 contrast ratio. Ensure icon colors have sufficient contrast against background colors with the [Colour Contrast Analyzer](https://www.tpgi.com/color-contrast-checker/).
+- Use color to enhance, not convey, icon meaning. For example, changing an icon's color from green to red to indicate a status change would fail accessibility requirements if no words or symbols were associated with the icon.
+
+<Whitespace />
+
+## Support Compliance
+
+Terra is committed to ensuring its components provide maximum possible accessibility. However, using Terra components will not automatically make your product accessible. At most, consuming this component accounts for 33% of a product’s accessibility — the rest is up to development teams to implement.
+
+This component has **[known / no known]** accessibility failures. If you find accessibility bugs or gaps, please [report these issues on Github](https://github.com/cerner/terra-core/issues/new/choose).
+
+#### Success Criteria
+
+_Icon **must** meet the following success criteria:_
+
+- [**1.3.1 Info and Relationships**](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html). - The intent of this Success Criterion is to ensure that information and relationships that are implied by visual or auditory formatting are preserved when the presentation format changes. For example, the presentation format changes when the content is read by a screen reader.
+- [**0.0.0. Criteria Name**](https://www.link-to-criteria.html) - copy main paragraph text for the criteria.
```

Comment:
I'm unable to open the https://www.link-to-criteria.html link.

Reply:
Good catch. That is just a placeholder. I will fix.

Reply:
I think the criteria on line 110 sufficient, so I removed line 111 in b6239ccb3c8cf0c60aa7a37abba0e8462b771cb1.